package.json

```
@@ -0,0 +1,131 @@
+{
+  "name": "hipo-frontend-ts-boilerplate",
+  "version": "0.0.0",
+  "description": "Hipo Frontend TypeScript Boilerplate",
+  "private": true,
+  "scripts": {
+    "build:release": "webpack --config webpack.release.config.js  --progress --colors",
+    "build:dev": "webpack --config webpack.dev.config.js --progress --colors",
+    "build:dev:live": "./node_modules/.bin/webpack-dev-server --config webpack.dev.config.js --progress --colors --watch",
+    "start": "npm run build:dev:live",
+    "type-check": "echo \"type-checking...\" && tsc --noEmit",
+    "type-check:watch": "npm run type-check -- --watch",
+    "eslint": "./node_modules/.bin/eslint --ext .jsx,.js,.tsx,.ts src/",
+    "eslint:fix": "./node_modules/.bin/eslint --fix --ext .jsx,.js,.tsx,.ts src/",
+    "stylelint": "echo \"stylelint-checking...\" && ./node_modules/.bin/stylelint 'src/**/*.{css,scss}'",
+    "stylelint:fix": "npm run stylelint -- --fix",
+    "prettify": "./node_modules/.bin/prettier --config ./.prettierrc.js --write 'src/**/*.{ts,tsx}'",
+    "lint": "npm run stylelint; npm run eslint",
+    "test": "./node_modules/.bin/jest",
+    "test:watch": "npm t -- --watch",
+    "changelog": "./node_modules/.bin/conventional-changelog --config ./node_modules/hipo-web-conventional-changelog/index.js --context ./changelog-context-variables.json -o CHANGELOG.md -r 0",
+    "checkPackageUpdates": "npx npm-check-updates"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Hipo/hipo-frontend-ts-boilerplate.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Hipo/hipo-frontend-ts-boilerplate/issues"
+  },
+  "devDependencies": {
+    "@babel/cli": "7.5.5",
+    "@babel/core": "7.5.5",
+    "@babel/plugin-proposal-class-properties": "7.5.5",
+    "@babel/plugin-proposal-object-rest-spread": "7.5.5",
+    "@babel/preset-env": "7.5.5",
+    "@babel/preset-react": "7.0.0",
+    "@babel/preset-typescript": "7.3.3",
+    "@hipo/eslint-config-base": "2.1.1",
+    "@hipo/eslint-config-react": "1.0.2",
+    "@hipo/stylelint-config-base": "1.0.0",
```

Can we use the latest version for these? What could go wrong?
```suggestion
    "@hipo/eslint-config-base": *,
    "@hipo/eslint-config-react": *,
    "@hipo/stylelint-config-base": *,
```

---

Yes we can